### DISCUSSION TITLE
Umijs 怎么enable jsx for jest的测试

### DISCUSSION THREAD

**Discussion by Katherineksl** [umijs/umi#12437] [Q&A]
写了一个jsx, 写jest测试的时候显示jsx is not enabled, tsx的没有报错
用generator自动生成Jest相关的配置的了

**Comment by 18777595220** [comment]
哥们，解决了吗？我也遇到单元测试的问题了

**Reply by Katherineksl** [reply]
解决了，有的同事用.js结尾写jsx了.. 改一下后缀为Jsx就好

**Comment by fz6m** [comment]
目前绝大多数项目都是 ts 的，只有很老旧的项目升级上来才可能存在 jsx ，所以对 jsx 的支持是不全面，可能存在问题的。

如有兴趣，可以帮助排查为什么存在此问题，或者考虑采用其他的单元测试方式，如 vitest 等。
